src/types.rs

```diff
@@ -147,7 +147,7 @@ impl Display for OscTimeError {
 #[cfg(feature = "std")]
 impl std::error::Error for OscTimeError {}
 
-/// see OSC Type Tag String: [OSC Spec. 1.0](http://opensoundcontrol.org/spec-1_0)
+/// see OSC Type Tag String: [OSC Spec. 1.0](http://opensoundcontrol.org/spec-1_0.html)
```

Comment:
The URL has changed in the meantime.

```suggestion
/// see OSC Type Tag String: [OSC Spec. 1.0](https://opensoundcontrol.stanford.edu/spec-1_0.html)
```